### DISCUSSION TITLE
colab free

### DISCUSSION THREAD

**Discussion by Efan3536** [anxety-solo/sd-webui-notebook#8] [Ideas]
I know how to use it on colab for free. How should I tell you?

**Comment by anxety-solo** [comment]
Hi. If you have a discord, describe your method [here](https://discord.gg/YPsmMwRzUV) is a temporary server.
If not, post the idea in this discussion if possible. Somehow :#

**Reply by Efan3536** [reply]
efan_3536，my discrod
